StatusForm.Designer.cs

```diff
@@ -196,11 +183,24 @@ private void InitializeComponent()
             this.label1.Location = new System.Drawing.Point(6, 92);
             this.label1.Margin = new System.Windows.Forms.Padding(6, 0, 6, 0);
             this.label1.Name = "label1";
-            this.label1.Size = new System.Drawing.Size(158, 27);
+            this.label1.Size = new System.Drawing.Size(202, 27);
             this.label1.TabIndex = 11;
             this.label1.Text = "Cache Folder:";
             this.label1.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
             // 
+            // CacheFolder
+            // 
+            this.CacheFolder.AutoSize = true;

```

Comment:
autosize?

Reply:
These files are auto generated, i actually made the changes visually in the interface builder..